README.md

```diff
@@ -63,7 +63,43 @@ print(result.text_content)
 docker build -t markitdown:latest .
 docker run --rm -i markitdown:latest < ~/your-file.pdf > output.md
 ```
+<details>
+    
+<summary>Batch Processing Multiple Files</summary>
 
+This example shows how to convert multiple files to markdown format in a single run. The script processes all supported files in a directory and creates corresponding markdown files.
+
+
+```python convert.py
+from markitdown import MarkItDown
+from openai import OpenAI
+import os
+client = OpenAI(api_key="your-api-key-here")
+md = MarkItDown(mlm_client=client, mlm_model="gpt-4o-2024-11-20")
```

Comment:
```suggestion
md = MarkItDown(llm_client=client, llm_model="gpt-4o-2024-11-20")
```

Reply:
Hello @gagb, this will not work. mlm parametr is define directly in the library's code. It can't be changed.

Reply:
It will also make your example consistent with the Python API above in the readme

Reply:
sorry, I was not aware you are going to change this parameters' terminology in the code